src/integration-test/java/uk/gov/ida/integrationtest/DefaultTrustStoreTest.java

```diff
@@ -63,17 +57,6 @@ public void prodTruststoresShouldContainCorrectCertificates() throws KeyStoreExc
                 .aliases()
         );
 
-        EidasMetadataConfiguration eidasaMetadataConfiguration = prodMsaWithDefaultTruststoresApplicationRule
```

Comment:
The default truststore test has been updated in this PR but the truststores themselves haven't. The truststores and their tests should be both updated at the same time so that the PR has internal consistency. So the change to the test needs to be taken out or the trustores update needs to be added to this PR.